index.js

```diff
@@ -5,6 +5,7 @@ class RPFormatter extends Formatter {
 
     constructor(options) {
         super(options);
+        if (!options.parsedArgvOptions.rpConfig.enable) return undefined;
```

Comment:
I think enable flag need to be true by default and disable intentionally setting false, otherwise it will be breaking change.